pkg/operatorsource/configuring.go

```diff
@@ -65,7 +58,7 @@ func (r *configuringReconciler) Reconcile(ctx context.Context, in *v1alpha1.Oper
 
 	out = in
 
-	cscName := getCatalogSourceConfigName(in.Name)
+	cscName := in.Name
```

Comment:
This is not required for the UI, but we are removing all prefixes, right?

Reply:
Correct. This is for more consistent naming when you look at the details or the command line.

Reply:
I will also add this to the commit message.